README.md

```diff
@@ -24,8 +24,6 @@
 
 # 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- [ OPTIONAL] Kindly replace the text [your_project_name] with the real name of the project you are working on
i.e `Vet Clinic`. With this you will have a professional readme file